isDeskOccupied.py

```diff
@@ -0,0 +1,303 @@
+import requests
+import argparse
+import sys
+import json
+import csv
+import time
+import os
+import tkinter
+from PIL import Image, ImageTk
+
+class isDeskOccupied:
+
+    #Set up workspace for API calls to Rhombus Systems
+    def __init__(self,args):
+        #Initialize argument parser
+        self.argParser = self.__initArgParser()
+        self.args = self.argParser.parse_args(args)
+        if self.args.dgui:
+            self.args.gui == False
+        #Create a session to set default call Header information
+        self.session = requests.session()
+        #self.session.headers = {"x-auth-scheme": "api-token","x-auth-apikey": self.args.apiKey}
+        self.session.headers = {"x-auth-scheme": "api-token","x-auth-apikey": self.args.apiKey}
+        #Set a default base URL for api calls
+        self.url = "https://api2.rhombussystems.com/api/"
+        self.time = time.time()
+        self.endTime = None
+        self.areas = []
+        self.humanMovementBounds = []
+
+    #Define arguments which the user may be prompted for
+    @staticmethod
+    def __initArgParser():
+        argParser = argparse.ArgumentParser(description = 'Determine wether defined workspaces are occupied or not on a camera by camera basis')
+        argParser.add_argument("apiKey", help = 'Your personal api key, get this from your Rhombus Console')
+        argParser.add_argument("uuid", help = 'UUID of target Camera')
+        argParser.add_argument("--newArea","--n", help = 'Input a string to define a new area bounding box, must include quotation marks eg: "(0,0)(5,10)"')
```

Comment:
are these areas defined in relation to the camera resolution or on a scale from [0, 1] normalized?  I would suggest all of the area points are normalized to [0,1] so that the user does not have to know what the resolution of the camera is.